Cybersuite children are now included in ATT pools

JoschiZ opened the pull request:
This is done by allowing the cybersuite to inherit attributes of their children and raising a ATT refresh when a cybersuite is added or removed. The relevant tooltips are updated to recursively add the children of Cybersuites.

fixes #5045

DelnarErsike commented:
I would honestly prefer a solution that is more general than this one. The main thing I'm worried about is that the fix specifically requires that (and relies on) the parent 'ware be of the "Cybersuite" category. This shouldn't necessarily have to be the case. Hardcoding around the "Cyberlimb" category already causes a few problems, we don't want to make this worse.

JoschiZ commented:
I extracted all but one check for "Cyberlimb" and replaced them by a single centralized property on the `Cyberware` itself, that checks if itself or any of it's children occupies any limb slots.
This should be functionally equivalent and will allow easier refactor in the future.
